bootstrap.sh

```diff
@@ -35,39 +35,47 @@ then
     then
         echo "Downloading clang-tidy and clang-format to bin directory"
```

Comment:
We do control the directory structure. I can change this to do that, at least in bootstrap.sh pretty easily, I think.